### DISCUSSION TITLE
my first integration/microcontroller project

### DISCUSSION THREAD

**Discussion by pfsequeira** [RealByron/PicoW-Intex-PureSpa#7] [General]
Hi,

First, I'd like to thank you for sharing your work. I'm starting to replicate your project and have already bought a Pico W. I'm still waiting for the other components to arrive. Have you encountered any issues during this time? Are you still using this project on a daily basis? This is my first experience with microcontrollers, and I have no background in electronics—my expertise is in computer science.

Best!

**Comment by jonpeltier11** [comment]


**Reply by RealByron** [reply]
Hi! 
I did fix the schematics, there was a missing 5v to power the RPi Pico

**Comment by RealByron** [comment]
> Hi,
> 
> First, I'd like to thank you for sharing your work. I'm starting to replicate your project and have already bought a Pico W. I'm still waiting for the other components to arrive. Have you encountered any issues during this time? Are you still using this project on a daily basis? This is my first experience with microcontrollers, and I have no background in electronics—my expertise is in computer science.
> 
> Best!

It's still working fine.
I just restart all of that to be that recent esphome version doesn't broke the project.

**Reply by koniu1983** [reply]
hello, it will works on esp8266? Like copy -> paste?
